xml/dm_registers.xml

```diff
@@ -615,6 +615,8 @@
         is halted or not. Unavailable/nonexistent harts are not considered to
         be halted.
 
+        This register may not be present in systems with fewer than 2 harts.
```

Comment:
"May not" is ambiguous, possibly meaning "shall not" and possibly meaning "may."  How about something like:
This register must be present in systems with 2 or more harts and may be in systems with 1 hart.

Also, is this statement related to the number of harts in the system or the number of harts attached to this DM?

Reply:
Does "might not" instead of "may not" work?

Reply:
That sounds good.

My other question still applies.  Should it be something like "This register might not be present if fewer than 2 harts are attached to the DM." or do you think it's correct as written?

Reply:
You are right on that point. I've pushed a change to reflect all this.